examples/identity-app/README.md

```diff
@@ -26,19 +26,46 @@ Be sure that Docker image von-network is on your Docker registry.
 If you have not the Docker image on your Docker registry, then build them.
 Please follow [Documentation](./images/von-network/README.md)
 ### Step 2
-Prepare a network.yaml file for your demo or you can use this sample for [AWS](../../platforms/hyperledger-indy/configuration/samples/network-indyv3-aries.yaml) or [Minikube](../../platforms/hyperledger-indy/configuration/samples/network-minikube-aries.yaml) and update for your case.
+Be sure that Docker image of agents is on your Docker registry.
+If you have not the Docker image on your Docker registry, then build them.
+Please follow [Documentation](./images/agents/README.md) 
 ### Step 3
+Prepare a network.yaml file for your demo or you can use this sample for [AWS](../../platforms/hyperledger-indy/configuration/samples/network-indyv3-aries.yaml) or [Minikube](../../platforms/hyperledger-indy/configuration/samples/network-minikube-aries.yaml) and update for your case.
+### Step 4
 Run Ansible Playbook by these [steps](../../platforms/hyperledger-indy/configuration/README.md)
 Can be started also via Docker container:
 1. Go to root directory of this git repository.
 2. Run command: `docker run -it -v $(pwd):/home/blockchain-automation-framework/ --entrypoint bash hyperledgerlabs/baf-build`
 3. When are you entered into Docker container console, use command:
 `./run.sh`
-### Step 4
-After Blockchain Automation Framework setup can be created Helm release of Indy Web Server:
-1. Create Helm release with command: `helm install --name authority-webserver ./charts/webserver/`
+### Step 5
+After Blockchain Automation Framework setup can be deployed Identity App via Ansible playbook:
```

Comment:
"After Indy network has been set up, Identity App can be deployed via Ansible playbook"